core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -120,6 +120,10 @@
       + "details, etc.";
   protected static final String METRICS_TAGS_DEFAULT = "";
 
+  public static final String SSL_KEYSTORE_RELOAD_CONFIG = "ssl.keystore.reload";
+  protected static final String SSL_KEYSTORE_RELOAD_DOC =
+      "Enable auto reload of ssl keystore";
+  protected static final boolean SSL_KEYSTORE_RELOAD_DEFAULT = false;
```

Comment:
this seems like a great addition. why do you want this default false?

Reply:
aren't your changes backwards compatible? 

i mean, i guess not if people are relying on the ability to change the keystore file and have nothing happen. but that seems like a crazy thing to do in the first place.

Reply:
revised, thanks!